pkgs/tools/misc/tmux/default.nix

```diff
@@ -1,10 +1,11 @@
 { stdenv, fetchurl, ncurses, libevent, pkgconfig }:
 
 stdenv.mkDerivation rec {
-  name = "tmux-2.0";
+  name = "tmux-${version}";
+  version = "2.0";
 
   src = fetchurl {
```

Comment:
If they're tarballs uploaded by the maintainer, then `fetchurl` should work fine. (At least I assume GitHub won't change those tarballs.)